src/react-components/ory/user-auth-card.tsx

```diff
@@ -290,6 +294,7 @@ export const UserAuthCard = ({
               data-testid={`${flowType}-flow`}
             >
               {$flow}
+              {showIdentifier && <IdentifierInfo flow={flow} />}
```

Comment:
Not sure if this belongs here, what happens if i only have a webauthn as a password?

Also this isn't part of the 2FA flow - even though it contains similar data.

Reply:
As we have `!isTwoFactor()` it's not triggering for 2FA flow, like it's reflected on Chromatic tests.

Reply:
ah okay nice!